src/DummyConsoleApp/RunProcessAsTask.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<doc>
+    <assembly>
+        <name>DummyConsoleApp</name>
+    </assembly>
+    <members>
+    </members>
+</doc>

```

Comment:
I think it's odd that this is committed.  Are you sure you want documentation in the first place?